crates/bevy_ui/src/lib.rs

```diff
@@ -110,6 +111,23 @@ impl Plugin for UiPlugin {
                 CoreStage::PostUpdate,
                 widget::image_node_system.before(UiSystem::Flex),
             )
+            .add_system_to_stage(
+                CoreStage::Update,
+                widget::update_slider_value.label(widget::UpdateSliderValue),
+            )
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                widget::update_slider_handle.after(widget::UpdateSliderValue),
+            )
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                widget::update_slider_value
+                    .label(widget::UpdateSliderValue),
+            )
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                widget::update_slider_handle.after(widget::UpdateSliderValue),
+            )
```

Comment:
`update_slider_handle` is added twice to the `PostUpdate` stage, which I'm pretty sure is incorrect

Reply:
Ditto